appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+# Test against the latest version of this Node.js version
+# environment:
+#   nodejs_version: "6"
+
+platform:
+  - x64
+
+cache:
+  - node_modules
+  - '%APPDATA%\npm-cache'
+
+init:
+  - git config --global core.autocrlf input
+
+# Install scripts (runs after repo cloning)
+install:
+  - ps: Install-Product node 6 x64
+  # - git reset --hard HEAD
+  - npm install
+  - npm prune
```

Comment:
I included this because of its use in numerous examples (occasionally placed above rather than below `npm install`), though I’ve not yet found any discussion or explanation of why/when it should be included.